src/xcode/ENA/ENA/Source/Developer Menu/Features/DMPosterGeneration/DMPosterGenerationViewController.swift

```diff
@@ -303,17 +303,17 @@ class DMPosterGenerationViewController: UIViewController, UITextFieldDelegate {
 		let pdfView = PDFView()
 		let pdfDocument = PDFDocument(data: templateData.template)
 
-		guard let qrCodeImage = viewModel.traceLocation.qrCode(size: CGSize(width: qrCodeSideLengthField.cgFloatValue, height: qrCodeSideLengthField.cgFloatValue), qrCodeErrorCorrectionLevel: .medium) else { return pdfView }
-		guard let hexColorString = descriptionFontColorField.text else { return pdfView }
+//		guard let qrCodeImage = viewModel.traceLocation.qrCode(size: CGSize(width: qrCodeSideLengthField.cgFloatValue, height: qrCodeSideLengthField.cgFloatValue), qrCodeErrorCorrectionLevel: .medium) else { return pdfView }
+//		guard let hexColorString = descriptionFontColorField.text else { return pdfView }
 
-		try? pdfDocument?.embedImageAndText(
-			image: qrCodeImage,
-			at: CGPoint(x: qrCodeOffsetXField.cgFloatValue, y: qrCodeOffsetYField.cgFloatValue),
-			text: viewModel.posterDescription,
-			of: descriptionFontSizeField.cgFloatValue,
-			and: UIColor().hexStringToUIColor(hex: hexColorString),
-			with: CGRect(x: descriptionOffsetXField.cgFloatValue, y: descriptionOffsetYField.cgFloatValue, width: descriptionWidthField.cgFloatValue, height: descriptionHeightField.cgFloatValue)
-		)
+//		try? pdfDocument?.embedImageAndText(
```

Comment:
Will we create a separate ticket to update the Developer Menu according to the new changes?

Reply:
Issue: https://jira-ibs.wbs.net.sap/browse/EXPOSUREAPP-6373

Reply:
Will take care of it.